### ISSUE TITLE
2.2.2 causes TypeError: Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by az33zy** [shakacode/sass-resources-loader#144]
Trying to build a Nuxt project. I get this error.

```
/node_modules/sass-resources-loader/lib/utils/rewritePaths.js:41
    if (single.indexOf('sass:') === 0) {
               ^

TypeError: Cannot read property 'indexOf' of undefined
    at .../node_modules/sass-resources-loader/lib/utils/rewritePaths.js:41:16
    at String.replace (<anonymous>)
    at rewritePaths (.../node_modules/sass-resources-loader/lib/utils/rewritePaths.js:39:28)
    at .../node_modules/sass-resources-loader/lib/loader.js:73:36
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:69:3)
error Command failed with exit code 1.
```

It works fine if I add this to `package.json`:

```
"resolutions": {
    "sass-resources-loader": "2.2.1"
  }
```

**Comment by mattiaerli97** [comment]
This error is really blocking and is a problem for the whole development of my company. 
**Can you merge the PR with absolute urgency**?

**Comment by hjujah** [comment]
**Critical** issue indeed...

**Comment by mattiaerli97** [comment]
> **Critical** issue indeed... Btw, for me it worked when I installed it explicitly as a dependency and version 2.2.1. (resolution didn't fix the issue)...

Yes I now, I downgraded the package locally, but for production I don't want to downgrade packages. However your workaround fixed the problem, at least temporarily

**Comment by Vlasterx** [comment]
I can confirm that this is happening to me as well. I had to downgrade back to 2.2.1

**Comment by justin808** [comment]
Shipped 2.2.3! 🚀
